### ISSUE TITLE
[minor] svg doesn't have .getElementById

### ISSUE THREAD

**Issue by caub** [tmpvar/jsdom#1938]
### Basic info:

node: 8.2.1
jsdom: 11.1.0

### Minimal reproduction case

```js
const { JSDOM } = require("jsdom");
const svg = new JSDOM(`
 <svg id="master-artboard" xmlns="http://www.w3.org/2000/svg">
  <g id="g"></g>
 </svg>
`).window.document.querySelector('svg');

console.assert(svg.getElementById)
```

### How does similar code behave in browsers?

```js
const svg = document.createRange().createContextualFragment(`
 <svg id="master-artboard" xmlns="http://www.w3.org/2000/svg">
  <g id="g"></g>
 </svg>
`).querySelector('svg');

console.log(svg.getElementById('g'))
```
works

-- 
I searched in the source where that function is attached, it's in Document-impl, or rather as NonElementParentNode

anyway it doesn't matter, above all in the context I'm using it, I use .querySelectorAll('#'+id) because those ids can be used multiple times

**Comment by caub** [comment]
sry didn't see

**Comment by caub** [comment]
will be fixed by #2011
